buildimage.go

```diff
@@ -65,7 +66,7 @@ func main() {
 			"-files", extraBinMust("wpa_action"),
 			"-files", extraBinMust("wpa_cli"),
 			"-files", extraBinMust("wpa_passphrase"),
-			"-files", "webboot/webboot:bbin/webboot")
+			"-files", goenv+"/src/github.com/u-root/webboot/webboot:bbin/webboot")
```

Comment:
if you are outside the gopath, why are you using a path that is *inside* the gopath. I'm lost.